aspnetcore/web-api/advanced/conventions.md

```diff
@@ -3,40 +3,60 @@ title: Use web API conventions
 author: pranavkm
 description: Learn about web API conventions in ASP.NET Core.
 monikerRange: '>= aspnetcore-2.2'
-ms.author: pranavkm
+ms.author: scaddie
 ms.custom: mvc
-ms.date: 11/13/2018
+ms.date: 11/30/2018
 uid: web-api/advanced/conventions
 ---
 # Use web API conventions
 
-ASP.NET Core 2.2 introduces a way to extract common [API documentation](xref:tutorials/web-api-help-pages-using-swagger) and apply it to multiple actions, controllers, or all controllers within an assembly. Web API conventions are a substitute for decorating individual actions with [[ProducesResponseType]](xref:Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute). It allows you to define the most common "conventional" return types and status codes that you return from your action with a way to select the convention method that applies to an action.
+By [Pranav Krishnamoorthy](https://github.com/pranavkm) and [Scott Addie](https://github.com/scottaddie)
 
-By default, ASP.NET Core MVC 2.2 ships with a set of default conventions, `Microsoft.AspNetCore.Mvc.DefaultApiConventions`. The conventions are based on the controller that ASP.NET Core scaffolds. If your actions follow the pattern that scaffolding produces, you should be successful using the default conventions.
+ASP.NET Core 2.2 introduces a way to extract common [API documentation](xref:tutorials/web-api-help-pages-using-swagger) and apply it to multiple actions, controllers, or all controllers within an assembly. Web API conventions are a substitute for decorating individual actions with [[ProducesResponseType]](xref:Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute).
 
-At runtime, <xref:Microsoft.AspNetCore.Mvc.ApiExplorer> understands conventions. `ApiExplorer` is MVC's abstraction to communicate with Open API document generators. Attributes from the applied convention get associated with an action and are included in the action's Swagger documentation. API analyzers also understand conventions. If your action is unconventional (for example, it returns a status code that isn't documented by the applied convention), it produces a warning, encouraging you to document it.
+A convention allows you to:
+
+* Define the most common return types and status codes returned from a specific type of action.
+* Identify actions that deviate from the defined standard.
+
+ASP.NET Core MVC 2.2 includes a set of default conventions in `Microsoft.AspNetCore.Mvc.DefaultApiConventions`. The conventions are based on the controller that ASP.NET Core scaffolds. If your actions follow the pattern that scaffolding produces, you should be successful using the default conventions.
```

Comment:
ASP.NET Core MVC 2.2 **and later** includes a set of default conventions

Reply:
> The conventions are based on the controller that ASP.NET Core scaffolds.

I don't understand that. That seems to contradict ASP.NET Core MVC 2.2 includes a set of default conventions in `Microsoft.AspNetCore.Mvc.DefaultApiConventions`.

Can you explain how scaffolding determines conventions? Why do you need to **follow the pattern that scaffolding produces**? Isn't that just a good example?